### PR TITLE
updates from upstream review of placement changes

### PR DESCRIPTION
**Pull request by gparvin**
Signed-off-by: Gus Parvin <gparvin@redhat.com>
Port review changes from community to this repo. https://github.com/open-cluster-management/backlog/issues/15604

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -137,6 +139,16 @@ func main() {
 	var generatedClient kubernetes.Interface = kubernetes.NewForConfigOrDie(mgr.GetConfig())
 	propagator.Initialize(cfg, &generatedClient)
 
+	cache := mgr.GetCache()
```
Comment (ckandag) [inline cmd/manager/main.go:142]: Hi Gus @gparvin , I did not understand where/what  this index function is used for.
Could you pls add a comment above just for one someone trying to understand the code later on.

everything else is good. thnx

Comment (gparvin) [reply cmd/manager/main.go:142]: Good point.  Thanks

Comment (gparvin) [reply cmd/manager/main.go:142]: @ckandag I just added a comment.  Can you look and see if it helps?

**Review by ckandag — APPROVED** [review]

**Comment by gparvin** [comment]
@ckandag Can you add the `/lgtm`  -- I had to make a change to force a rebuild that wasn't a comment for some reason.

**Pull request merged**